plan/planbuilder.go

```diff
@@ -1051,23 +1051,28 @@ func composeShowSchema(names []string, ftypes []byte) *expression.Schema {
 		col := &expression.Column{
 			ColName: model.NewCIStr(name),
 		}
-		var retType types.FieldType
+		var retTp byte
```

Comment:
😂😂 retTp is `FieldType.Tp`, and retType is `FieldType`. retTp and retType itself is not the source of confusing.. We may refine the naming later..